### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

**Comment by Nekotekina** [comment]
Now there is a significant change. Report if something broke.

**Comment by TCVinNYC** [comment]
LLVM files were rendered in a different folder.
image
image

**Comment by Zangetsu38** [comment]
ah yes, i have same think, now obj llvm is stocked all in root rpcs3 folder

**Comment by TCVinNYC** [comment]
does it affect performace tho?
if not i think its best to just have it in a folder

**Comment by Zangetsu38** [comment]
also, i have other strange think, after re launch game, i have again one module compilling

**Comment by Luffykun007** [comment]
For me also the same happens!

capturar

**Comment by Nekotekina** [comment]
Thanks for reporting!

**Comment by Couls** [comment]
image
Black screen in demon souls after intro cutscene
E {PPU[0x1000005] Thread (HighGraphics) [0x01d2c0c4]} LDR: Already linked function 'cellPamfStreamTypeToEsFilterId' in module 'cellPamf'
F {PPU[0x1000005] Thread (HighGraphics) [0x01d2c0c4]} MEM: Access violation writing location 0x22f906c

**Comment by Luffykun007** [comment]
Same mistake for me in Demons souls after intro cutscene.

capturar


U {PPU[0x100001c] Thread (_libsail-control) [0x01d3d038]} sys_prx TODO: _sys_prx_unload_module(id=0x230f7d00, flags=0x0, pOpt=*0x0)
U {PPU[0x100001c] Thread (_libsail-control) [0x01d3d038]} sys_prx TODO: _sys_prx_unload_module(id=0x230f7c00, flags=0x0, pOpt=*0x0)
U {PPU[0x1000005] Thread (HighGraphics) [0x01d3d038]} sys_prx TODO: _sys_prx_unload_module(id=0x230f6400, flags=0x0, pOpt=*0x0)
U {PPU[0x1000005] Thread (HighGraphics) [0x01d3d038]} sys_prx TODO: _sys_prx_unload_module(id=0x230f6300, flags=0x0, pOpt=*0x0)
E {PPU[0x1000005] Thread (HighGraphics) [0x01d3c9ac]} LDR: Already linked function 'cellPamfStreamTypeToEsFilterId' in module 'cellPamf'
F {PPU[0x1000005] Thread (HighGraphics) [0x01d3c9ac]} MEM: Access violation writing location 0x220c06c

**Comment by Zangetsu38** [comment]
i have already signaled this problem in discord, not caused by this PR

**Comment by Zangetsu38** [comment]
for module llvm compile it is good now :)

**Comment by Zangetsu38** [comment]
broken Minecraft for me, and stubbed not really fix problem, first image freeze
>F {PPU[0x1000000] Thread (main_thread) [0x01b63930]} class std::runtime_error thrown: Unimplemented
(in file C:\rpcs3\rpcs3\Emu\Cell\Modules\sys_libc_.cpp:300)

**Comment by SakataGintokiYT** [comment]
@Nekotekina 
**Regression by this PR:**
[58f43db7]
**Log:** https://gist.github.com/SakataGintokiYT/ae5b6fb89f5c705e91c41d21b808db70
**Demo - One Piece : Kaizoku Musou 2**
`F {PPU[0x1000030] Thread (_libsail-control) [0x008d70c4]} MEM: Access violation reading location 0xea5570`

bez tytulu

**Comment by Nekotekina** [comment]
Test with liblv2 and without, please.

**Comment by SakataGintokiYT** [comment]
Minecraft work now with libv2 and auto required libraries.

**Comment by Zangetsu38** [comment]
Yes, same in my build now, works in lv2 and without :)

**Comment by Zangetsu38** [comment]
Yes, same in my build now, works in lv2 and without :)
I have restored libssl and libhttp for test in lv2 and works, problem caused by only librudp.
I think you can remove this, and ignore only librudp ^^, ssl required by few game for boot :)

**Comment by Zangetsu38** [comment]
you can try fix problem sys_fs with Wipeut HD PSN with last Update please, i have loop
>·W 0:00:02.190596 {PPU[0x1000010] Thread (fios mediathread) [0x0145bee4]} sys_fs: sys_fs_stat(path=“”, sb=*0xd00d7a28)
·W 0:00:02.190635 {PPU[0x1000010] Thread (fios mediathread) [0x0145bee4]} vfs::get(): invalid input: 
·W 0:00:02.190641 {PPU[0x1000010] Thread (fios mediathread) [0x0145bee4]} sys_fs: sys_fs_stat(“”) failed: not mounted
·E 0:00:02.190645 {PPU[0x1000010] Thread (fios mediathread) [0x0145bee4]} 'sys_fs_stat' failed with 0x8001003a : CELL_ENOTMOUNTED

**Comment by Zangetsu38** [comment]
I have re found game, it is the darkness.
log with last build, rthis game no have update, and demo have same error.
[The Darkness.txt](https://github.com/RPCS3/rpcs3/files/1142339/The.Darkness.txt)

File: rpcs3/rpcs3qt/game_list_frame.cpp [hunk 1]
```diff
@@ -565,6 +565,7 @@ void game_list_frame::doubleClickedSlot(const QModelIndex& index)
 		{
 			LOG_SUCCESS(LOADER, "Boot from gamelist per doubleclick: done");
 			RequestAddRecentGame(q_string_pair(qstr(Emu.GetBoot()), qstr("[" + m_game_data[i].info.serial + "] " + m_game_data[i].info.name)));
+			Refresh(true);
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/game_list_frame.cpp]: why refresh?

and why from drive?

**Comment by raveskirza** [comment]
After this PR. There is some odd behavior with how the game configuration is read.
e.g. If you have the PS3_GAME (i.e. contents of disc) under /dev_hdd0/disc/BLES00296/
And if you have the USRDIR folder with EBOOT.BIN for that game under /dev_hdd0/game/BLES00296/
When you boot from the game list, the global config.yml is used instead of the game config.yml under data/BLES00296/config.yml

This is how the Title in the GSframe looks and the configuration is read from the config.yml from the root folder instead of the per game config.yml stored in /data/<Game Title>

image

**Comment by Zangetsu38** [comment]
@raveskirza i don't have this me, for all game have update, rpcs3 using good config setting for this game, and no global.

**Pull request merged**